examples/signalQuality.js

```diff
@@ -0,0 +1,9 @@
+const { createEEG, epoch, addSignalQuality } = require("..");
+const { tap, map } = require("rxjs/operators");
+
+const eeg1$ = createEEG().pipe(
```

Comment:
After our discussion, I'll change the pipe to output just the signal quality, thus letting us remove `add` from the name.

Still uncertain whether to make the signal quality data an array or an a object. Object is better from a JS engineering perspective, but if we make it an array it'll be more similar to the format of the `data` in Epochs and Samples and also make it more straightforward to use without having to have `channelNames` in the info property

Reply:
What about an object with the index of the channel as the key? Users can always do `Object.values(signalQuality)` to convert it to an array.

Reply:
So, after some thinking, I've decided that it makes the most sense to keep EEG data and signal quality info together in the same observable and thus stick with the 'addSignalQuality' operator that adds the signalQuality property to the Epoch object.

At least that's what is most convenient for my app, where I want to have components recording or plotting raw EEG data alongside some graphical representation of signal quality.

However, I have changed the signalQuality property to an object. It will either use keys from info.channelNames or just the channel index

Reply:
Thanks for the updates! I still think we should have separate signal "side effect" streams from the data stream, but let's go ahead and merge this, test, and iterate.